src/main/java/com/sonyericsson/rebuild/RebuildAction.java

```diff
@@ -67,6 +68,7 @@
 public class RebuildAction implements Action {
 
     private static final String SVN_TAG_PARAM_CLASS = "hudson.scm.listtagsparameter.ListSubversionTagsParameterValue";
+    private static final String NODE_LABEL_PARAM_CLASS = "org.jvnet.jenkins.plugins.nodelabelparameter.LabelParameterValue";
```

Comment:
there is also `org.jvnet.jenkins.plugins.nodelabelparameter.NodeParameterValue`, which extends `org.jvnet.jenkins.plugins.nodelabelparameter.LabelParameterValue`